cpu/native/periph/can.c

```diff
@@ -20,7 +20,7 @@
  */
 
 #if !defined(__linux__)
-#error "MODULE can_linux is only available on Linux"
+#error "MODULE periph_can is only available on Linux"
```

Comment:
I currently can't check, but OTOMH this is not correct. 
Many controllers (stm32 for example) also offer a can peripheral.

Reply:
This error is in the `native` can implementation...

Reply:
Yes, but since the native can implementation now uses the same can implementation as can peripherals (periph_can), the statement "MODULE periph_can is only available on Linux" doesn't sound right?

Reply:
This message is there for when trying to use the `periph_can` on a native instance that is not linux (e.g. on OSX), although I'd say that in that case the features should not even be there. Keep in mind that this file is `cpu/native` specific.

> the statement "MODULE periph_can is only available on Linux" doesn't sound right?

How do you propose to reword this?

Reply:
I understand the purpose of this message (even though at first I also misinterpreted it), but for people unfamiliar to RIOT or unfamiliar to the RIOT can-system, this message is very ambiguous. It can be understood as "You can only use periph_can when running RIOT on Linux (i.e. native)", which is not the case. 
I would reword it to something along the lines of "Usage of periph_can on RIOT native requires libsocketcan. Currently this is only available on Linux". 

(I'm still not 100% convinced about my own proposition. There's still room for confusion. But I would go more or less in this direction to avoid users from misinterpreting what is supported and what is not.)

Reply:
I've adapted the message